make/osde2e.mk

```diff
@@ -0,0 +1,18 @@
+DIR := $(dir $(realpath $(firstword $(MAKEFILE_LIST))))
+MANAGED_API_TEST_HARNESS_IMAGE ?= $(REG)/$(ORG)/managed-api-test-harness:osde2e
+
+.PHONY: image/osde2e/build
+image/osde2e/build:
+	go mod vendor
+	docker build . -f Dockerfile.osde2e -t $(MANAGED_API_TEST_HARNESS_IMAGE)
+
+.PHONY: image/osde2e/push
+image/osde2e/push:
+	docker push $(MANAGED_API_TEST_HARNESS_IMAGE)
+
+.PHONY: image/osde2e/build/push
+image/osde2e/build/push: image/osde2e/build image/osde2e/push
```

Comment:
These make targets can stay if they are needed, but they won't be used to build and push the images via CI. The new image and mirroring should be added to the release repo instead and OpenShift CI will deal with building and syncing the images for you.

https://github.com/openshift/release/blob/master/ci-operator/config/integr8ly/integreatly-operator/integr8ly-integreatly-operator-master.yaml#L17

```
- dockerfile_path: Dockerfile.osde2e
  from: os
  inputs:
    openshift_release_golang-1.13:
      as:
      - registry.svc.ci.openshift.org/openshift/release:golang-1.13
  to: integreatly-operator-test-harness-osde2e
```

https://github.com/openshift/release/blob/master/core-services/image-mirroring/integr8ly/mapping_integr8ly_operator#L2

```
registry.svc.ci.openshift.org/integr8ly/integreatly-operator:integreatly-operator-test-harness-osde2e quay.io/integreatly/managed-api-test-harness:osde2e
```

Note: This PR will need to be merged before you can merge the PR back with this config to the release repo

Reply:
Thanks a lot @mikenairn, that's great!